### PR TITLE
feat: add a prisma graphql-generator

### PR DESCRIPTION
**Pull request by maxmousse**

### CODE REVIEW

File: libs/hapify/generators/graphql-resolvers/src/lib/generators/module-definition/module.generator.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export function generateModuleDefinitionSourceFile(
+  project: Project,
+  path: string,
+) {
+  const fileName = `graphql.module-definition.ts`;
+  const filePath = `${path}/${fileName}`;
+
+  const sourceFile = project.createSourceFile(filePath, undefined, {
+    overwrite: true,
```
Comment (Mickael-Martos) [inline libs/hapify/generators/graphql-resolvers/src/lib/generators/module-definition/module.generator.ts]: Do you still want to use the overwrite for the module.generator ?

File: libs/hapify/prisma/nestjs-graphql-resolvers-generator/src/generator.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        fs.writeFileSync(
+          path.join(outputDirectory, 'hapify.json'),
+          inspect(schema, true, 10),
+        );
+      } catch (error) {
+        logger.error(error);
+        throw error;
+      }
+
+      // Remove unused imports
```
Comment (Mickael-Martos) [inline libs/hapify/prisma/nestjs-graphql-resolvers-generator/src/generator.ts:105]: Do you need this commented part for later ?

File: libs/hapify/generators/graphql-resolvers/src/lib/generators/module-definition/module.generator.ts (hunk 1 above)
Comment (maxmousse) [reply libs/hapify/generators/graphql-resolvers/src/lib/generators/module-definition/module.generator.ts]: No indeed, I fixed it!

File: libs/hapify/prisma/nestjs-graphql-resolvers-generator/src/generator.ts (hunk 2 above)
Comment (maxmousse) [reply libs/hapify/prisma/nestjs-graphql-resolvers-generator/src/generator.ts:105]: Yes we can remove the unused imports after reviewing the generated code, or if there is unwanted imports, we won't be able to see them.

**Pull request merged**